pkg/fs/config.go

```diff
@@ -70,11 +40,11 @@ func getFSMultipartsSessionConfigPath() (string, *probe.Error) {
 	if customMultipartsConfigPath != "" {
 		return customMultipartsConfigPath, nil
 	}
-	u, err := userCurrent()
-	if err != nil {
-		return "", err.Trace()
+	homeDir, e := user.HomeDir()
+	if e != nil {
+		return "", probe.NewError(e)
 	}
-	fsMultipartsConfigPath := filepath.Join(u.HomeDir, ".minio", "multiparts-session.json")
+	fsMultipartsConfigPath := filepath.Join(homeDir, ".minio", "multiparts-session.json")
 	return fsMultipartsConfigPath, nil
 }
 
```

Comment:
FS should never access config code. Apart from that,  I don't see a need for getFSBucketsConfigPath itself. Any path FS needs to deal with, should be relative to its root path.